gd/memory/interface.py

```diff
@@ -493,6 +493,18 @@ def enable_object_limit(self) -> None:
         self.write_bytes(Buffer[0x80, 0x38, 0x01, 0x00], 0x87B17)
         self.write_bytes(Buffer[0x80, 0x38, 0x01, 0x00], 0x880F4)
         self.write_bytes(Buffer[0x80, 0x38, 0x01, 0x00], 0x160B06)
+    def enable_anticheat(self) -> None:
```

Comment:
Everything that is after `;` should be moved into another function - `disable_anticheat`.